OpenRA.Mods.Common/Traits/World/PathFinder.cs

```diff
@@ -33,7 +33,7 @@ public interface IPathFinder
 		/// Calculates a path for the actor from source to destination
 		/// </summary>
 		/// <returns>A path from start to target</returns>
-		List<CPos> FindUnitPath(CPos source, CPos target, Actor self);
+		List<CPos> FindUnitPath(CPos source, CPos target, Actor self, Actor ignoredActor);
```

Comment:
Please use `ignoreActor` instead of `ignoredActor` for consistency with the rest of the code.